microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+
+|   | What Setting Edge uses? |
+|---|-------------------------|
+| Language used for rendering Edge’s UI | edge://settings/languages. One item in the “Languages” list will indicate “Microsoft Edge is displayed in this language” |
+| Language used to render web sites | **Website Server-side processing**: Edge sends the language list (as regional codes) specified in edge://settings/languages via the accept-language header. <br><br> **Website Client-side processing**: Edge makes the language list (as regional codes) available via the navigator.languages DOM API. The preferred language (at the top of the list) is also available as navigator.language (note: singular) |
+| Language/region used for formatting date, time, numbers, etc. | The language in the edge://settings/languages list specified as the “displayed language” |
+
+While many OSes supported by Microsoft Edge do have a facility to specify regional information preferences, this OS information (as a regional code) is not used by the browser, and customers who make these changes to the OS have no similar mechanism to express these regional preferences in Microsoft Edge (except to change their display language and region). 
+
+## Opportunity
+In order to better reflect a customer’s regional preferences (especially in the cases where the customer has changed their default preferences in the OS), Microsoft Edge is making a new Setting available in the “Language” section of Settings:
+
+![Screenshot of Edge setting UI titled "Share additional OS regional format" with the option "limited (default)" selected.](media/os-regional-setting-screenshot.msft.png)
+
+This setting allows the customer’s OS regional preferences to be automatically made available to websites under “limited” cases by default (more on that below), and offers two other options, including “never” which reverts the feature back to legacy behavior. 
+
+This additional sharing will allow websites to have visibility into more customer’s specific regional preferences than was previously possible. Customers that changed their OS regional preferences may now see their preferences reflected automatically in more websites. Previously, these users preferences would not be visible to websites, and their [incorrect] preferences would be assumed from the browser’s display language default region instead. 
+
+|    | What Setting Edge uses? |
+|----|-------------------------|
+| Language/region used for formatting date, time, numbers, etc. | ~~The language in the edge://settings/languages list specified as the “displayed language”~~<br><mark>The language/region specified in the OS settings*</mark> |
```

Comment:
i'll FTS tilde and consider moving * note into table cell

Reply:
I did an editing pass.  Not sure what you meant, but I cleaned it up